pkg/deployment/config.go

```diff
@@ -61,6 +61,12 @@ const (
 	queueSidecarCPULimitKey              = "queueSidecarCPULimit"
 	queueSidecarMemoryLimitKey           = "queueSidecarMemoryLimit"
 	queueSidecarEphemeralStorageLimitKey = "queueSidecarEphemeralStorageLimit"
+
+	// concurrencyStateEndpointKey is the key to configure the endpoint for acting when traffic drops to / increases from zero.
+	concurrencyStateEndpointKey = "concurrencyStateEndpoint"
+
+	// ConcurrencyStateEndpointDefault is the default endpoint for acting when traffic drops to / increases from zero.
+	concurrencyStateEndpointDefault = ""
```

Comment:
I dont know if we really need a constant for this at all tbh (I'd just do it inline like we do with RegistriesSkippingTagResolving), but if we _do_ I think the comment is a bit unclear, especially since the default being set is "" (i.e. no endpoint at all).